meta/src/local.rs

```diff
@@ -36,6 +35,13 @@ pub struct LocalTask {
     pub title: String,
 }
 
+/// Return a closure that checks if a package has a proc-macro or dylib target.
+fn is_dylib_or_proc_macro() -> Box<dyn Fn(&cargo_metadata::Target) -> bool> {
```

Comment:
Ah, I meant a function `fn is_dylib_or_proc_macro(target: &cargo_metadata::Target) -> bool`.

Reply:
This way is wide better, thanks for the review. o/